components/webhook-mgt/org.wso2.carbon.identity.webhook.management/src/main/java/org/wso2/carbon/identity/webhook/management/api/core/cache/WebhookCacheEntry.java

```diff
@@ -19,7 +19,7 @@
 package org.wso2.carbon.identity.webhook.management.api.core.cache;
 
 import org.wso2.carbon.identity.core.cache.CacheEntry;
-import org.wso2.carbon.identity.webhook.management.api.model.Webhook;
+import org.wso2.carbon.identity.webhook.management.api.model.webhook.Webhook;
```

Comment:
Normally we don't categorize model classes for different packages

Reply:
fixed with https://github.com/wso2/carbon-identity-framework/pull/6885/commits/fb575c056228c93d2221b4ea29c500a76d09311c